sphinx/writers/html.py

```diff
@@ -174,10 +174,10 @@ def depart_desc_type(self, node: Element) -> None:
         pass
 
     def visit_desc_returns(self, node: Element) -> None:
-        self.body.append(' &#x2192; ')
+        self.body.append(' <span class="sig-return"><span class="sig-return-icon">&#x2192;</span> <span class="sig-return-typehint">')
```

Comment:
Could you fold a line, please? (or separate appending to multiple calls)
Our flake8 warns this line (also html5.py).
https://github.com/sphinx-doc/sphinx/pull/9225/checks?check_run_id=2591018763